lib/irq.c

```diff
@@ -380,15 +380,12 @@ static int
 validate_irq_subindex(vfu_ctx_t *vfu_ctx, uint32_t subindex)
```

Comment:
now this doesn't fiddle with errno, can you make it a boolean return like other validate functions?

Reply:
Sure